api/src/shared/infrastructure/repositories/user-repository.js

```diff
@@ -279,9 +264,9 @@ const updateUserDetailsForAdministration = async function ({
 };
 
 const updateHasSeenAssessmentInstructionsToTrue = async function (id) {
-  const user = await BookshelfUser.where({ id }).fetch({ require: false });
-  await user.save({ hasSeenAssessmentInstructions: true }, { patch: true, method: 'update' });
-  return bookshelfToDomainConverter.buildDomainObject(BookshelfUser, user);
+  const [user] = await knex('users').where({ id }).update('hasSeenAssessmentInstructions', true).returning('*');
```

Comment:
Bookshelf pouvait se charger de modifier automatiquement le `updatedAt` d'un enregistrement, mais avec Knex c'est à faire manuellement à chaque requête effectuant une modification :

```suggestion
  const [user] = await knex('users').where({ id }).update({
    hasSeenAssessmentInstructions: true,
    updatedAt: new Date(),
  ).returning('*');
```